runtime/agung/src/lib.rs

```diff
@@ -400,6 +349,10 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = System;
 	type WeightInfo = ();
+	type FreezeIdentifier = (); // TODO
```

Comment:
I saw some TODO here. Sudo we change it? Or that's okay we keep this.

Reply:
Will check it again 👀

Reply:
Removed the comments, after I checked the new possible features... I think we don't need this at the moment :+1: